src/store/index.js

```diff
@@ -0,0 +1,25 @@
+import thunk from "redux-thunk";
+
+import { createStore, applyMiddleware } from "redux";
+import { persistStore, persistReducer } from "redux-persist";
+import storage from "redux-persist/lib/storage";
+
+import rootReducer from "./reducers";
+
+const initialState = {};
+const persistConfig = {
+  key: "root",
+  storage,
+};
+
+const persistedReducer = persistReducer(persistConfig, rootReducer);
+
+export default () => {
+  let store = createStore(
+    persistedReducer,
+    initialState,
+    applyMiddleware(thunk)
+  );
+  let persistor = persistStore(store);
+  return { store, persistor };
```

Comment:
Okay, you use react-persist. But where is the gate for it? It should be at the top level of the application, near the store provider, but I can't find it.